tests/test_core.py

```diff
@@ -0,0 +1,74 @@
+from obr.core.core import get_mesh_stats
+from pathlib import Path
+import pytest
+
+
+@pytest.fixture
+def create_ansa_owner(tmpdir):
+    fn = "owner"
+    owner_content = """/*--------------------------------------------------------------------------------------------*\
+|                                                                                              |
+|    ANSA_VERSION: 22.1.0                                                                      |
+|                                                                                              |
+|    file created by  A N S A  Wed Mar  9 13:20:26 2022                                        |
+|                                                                                              |
+|    Output from: /home/mockett/Projects_TechActive/AutoCFD-3/Case01/GridConversion/c1g1.ccm   |
+|                                                                                              |
+\\*--------------------------------------------------------------------------------------------*/
```

Comment:
Can this change frequently? Is a regex maybe a more stable option like with the OpenFOAM headers?